package.json

```diff
@@ -4,8 +4,9 @@
   "description": "Source for building https://dist.ipfs.io.",
   "main": "",
   "scripts": {
-    "lint": "standard scripts/*.js site/assets/js/*.js && shellcheck *.sh",
-    "clean": "rm -f site/static/*.js site/static/*.css",
+    "standard": "standard scripts/*.js site/assets/js/*.js",
+    "lint": "npm run standard && shellcheck *.sh",
+    "clean": "rm -rf site/static/*.js site/static/*.css site/data/releases",
```

Comment:
the `dist` target adds data files to `site/data/releases` but `clean` wasn't cleaning it, so now it does.